Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -103,6 +107,14 @@ public void initialize() {
         servoGrabber = (ServoImplEx) hardwareMap.servo.get("servoGrabber");
         servoGrabber.setPwmRange(maxRange);
 
+        // limit switch
+        limitSwitch = hardwareMap.get(DigitalChannel.class, "limitSwitch");
+        limitSwitch.setMode(DigitalChannel.Mode.INPUT);
+
+        try {
+            blinkinChassis = (RevBlinkinLedDriver) hardwareMap.get(RevBlinkinLedDriver.class, "blinkinChassis");
+        } catch (Exception e) {}
```

Comment:
You shouldn't catch and swallow all exceptions

Reply:
not sure what this should be changed to, this section of code appeared when I merged master into this branch to fix merge conflicts

Reply:
omg why is that in master that's not good. You can just remove the try catch and just have the inside part. It should throw an exception if it fails.
```suggestion
        blinkinChassis = (RevBlinkinLedDriver) hardwareMap.get(RevBlinkinLedDriver.class, "blinkinChassis");
```